### DISCUSSION TITLE
Release Notes - v9.1.3

### DISCUSSION THREAD

**Discussion by vetrivelak** [visjs/vis-network#1964] [Q&A]
Hi Team, 
    v9.1.3 is not available in the release notes. Is there any specific reason for it? 

Thanks,
Vetrivel K.

**Comment by Thomaash** [comment]
Hello, the reason is probably just some issue with CI (crash, out of credits; the logs are not stored forever and are gone by now). I don't think it was ever deployed beyond receiving a git tag.

If you or anyone else not just curious and actually need the version deployed, feel free to reopen.
